app/models/miq_expression.rb

```diff
@@ -1775,15 +1775,9 @@ def self.atom_error(field, operator, value)
                                                                                     :error_message => err.message}
       end
       return false
-    when :ruby
-      return _("Ruby Script must not be blank") if value.blank?
-      return false
```

Comment:
right, ruby expression are not supported. there is error thrown in self._to_ruby method